evap/staff/templates/staff_semester_view.html

```diff
@@ -23,31 +23,39 @@
     {% if num_courses > 0 %}
     <div class="panel panel-info">
         <div class="panel-heading">
-            <h3 class="panel-title">{% trans "Overview" %}</h3>
+            <a class="accordion-toggle" data-toggle="collapse" href="#accordion" onclick="saveCollapseState()">
+                <h3 class="panel-title">{% trans "Overview" %}</h3>
```

Comment:
please allow collapsing when clicking on the arrow as well (have a look at the FAQ pages)